src/Callbacks/ForbiddenFilesConditions.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EcEuropa\Toolkit\Callbacks;
+
+use EcEuropa\Toolkit\TaskRunner\Commands\ToolCommands;
+
+class ForbiddenFilesConditions
+{
+
+    public static function grumpPhpisNotInstalled() {
+        // If grumphp package is not present in a project, then grumphp config file must not be present.
```

Comment:
This comment can be moved into the method Docblock

Reply:
Hi @joaocsilva this has been fixed.